### PR TITLE
[Bugfix] Fix header overlay being offscreen

### PR DESCRIPTION
**Pull request by awy3**
This is for this bug

https://trello.com/c/YSdveJrl/1021-ui-releated-bug-signature-overlay-is-off

We prevent the header from having a left offset that is negative so it won't go offscreen

### CODE REVIEW

File: src/helpers/getOverlayPositionBasedOn.js [hunk 1]
```diff
@@ -39,5 +39,5 @@ export default (element, overlay, align = 'left') => {
     right = 'auto';
     left = buttonLeft - (overlayWidth - buttonWidth);
   }
-  return { left, right };
+  return { left: Math.max(left, 0), right };
```
Comment (ZhijieZhang) [inline src/helpers/getOverlayPositionBasedOn.js:42]: Do you need to do the same thing for `right` to make sure it won't be cut off on the right side of the bounding box?

Comment (awy3) [reply src/helpers/getOverlayPositionBasedOn.js]: I tested this more and found a bug that I fixed

As for `right`, I don't think we need anything. The `right` get set to "auto" or "16" so it doesn't seem to overflow. It's mainly just `left`that get set to a negative number sometime and appear off screen

File: src/helpers/getOverlayPositionBasedOn.js [hunk 2]
```diff
@@ -39,5 +39,10 @@ export default (element, overlay, align = 'left') => {
     right = 'auto';
     left = buttonLeft - (overlayWidth - buttonWidth);
   }
-  return { left, right };
+
+
+  return { 
+    left: !isNaN(left) ? Math.max(left, 0) : left,
```
Comment (awy3) [inline src/helpers/getOverlayPositionBasedOn.js:45]: left can be set to "auto"

File: src/helpers/getOverlayPositionBasedOn.js (hunk 1 above)
Comment (ZhijieZhang) [reply src/helpers/getOverlayPositionBasedOn.js]: I see, thanks for the clarification!

**Pull request merged**